tasties_app/templates/tasties_app/base.html

```diff
@@ -0,0 +1,79 @@
+{% load static %} {% load bootstrap_icons %}
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js"></script>
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.min.js"
+      integrity="sha384-Atwg2Pkwv9vp0ygtn1JAojH0nYbwNJLPhwyoVbhoPwBhjQPR5VtM2+xf0Uwh9KtT"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" type="text/css" href="{% static 'css/base.css' %}" />
+    <title>tasties</title>
+  </head>
+  <body>
+    <!-- Navbar -->
+    <nav class="navbar navbar-light bg-light">
+      <a class="navbar-brand" style="color: #10654e">
+        <img
+          src="{% static 'images/logo.png' %}"
+          width="30"
+          height="25"
+          class="d-inline-block align-top"
+          alt="tasties-logo"
+          style="margin-top: 5px"
+        />
+        <strong style="font-size: 25px">tasties</strong>
+      </a>
+      <div class="form-group" style="position: relative; top: 10px">
+        <span class="input-group-append">
+          <input
+            type="text"
+            class="form-control"
+            placeholder="Search"
+            style="width: 350px; border-radius: 20px"
+            id="search"
+          />
+          <button
+            class="btn btn-outline-secondary bg-white border-start-0 border rounded-pill ms-n3"
+            style="position: relative; left: -40px"
+            id="search-button"
+          >
+            {% bs_icon 'search' %}
+          </button>
+        </span>
+      </div>
+      <div class="dropdown">
+        <button
+          class="btn btn-disabled dropdown-toggle"
+          type="button"
+          data-bs-toggle="dropdown"
+        >
+          {% bs_icon 'person-fill' %} Hello, username
+        </button>
+        <div class="dropdown-menu" aria-labelledby="dropdownMenuLink">
+          <a class="dropdown-item" href="#">Add recipe</a>
+          <a class="dropdown-item" href="#">Logout</a>
+        </div>
+      </div>
+    </nav>
+    <!-- Navbar -->
+
+    {% block content %}{% endblock %}
+
+    <!-- footer -->
+    <div class="footer flex-container bg-light fixed-bottom" id="footer">
+      <p>tasties @ 2022</p>
```

Comment:
I think we would want a '©' symbol here instead of the '@'.

Reply:
Thanks. Resolved.